notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue

```diff
@@ -105,13 +109,17 @@ export default {
     }
   },
   computed: {
-    unreadCount() {
-      return this.notifications.filter(
-        ({ attributes }) => !JSON.parse(attributes?.READ?.[0] || "true")
-      ).length;
+    count() {
+      if (this.countAllNotifications) {
+        return this.notifications.length;
+      } else {
+        return this.notifications.filter(
+          ({ attributes }) => !JSON.parse(attributes?.READ?.[0] || "true")
+        ).length;
+      }
     },
     variant() {
-      return this.unreadCount < 1 ? "default" : "danger";
+      return this.count < 2 ? "default" : "danger";
```

Comment:
:question: Why is having a single notification ignored from the stylistic highlight?

Reply:
Great catch! That was cut&paste regression